entity/templates/src/main/webapp/app/_entity-controller.js

```diff
@@ -45,23 +45,25 @@ angular.module('<%=angularAppName%>')
 
         $scope.delete = function (id) {
             <%= entityClass %>.get({id: id}, function(result) {
-                $scope.<%= entityInstance %> = result;
-                $('#delete<%= entityClass %>Confirmation').modal('show');
-            });
-        };
 
-        $scope.confirmDelete = function (id) {
-            <%= entityClass %>.delete({id: id},
-                function () {
-                    <%_ if (pagination != 'infinite-scroll') { _%>
-                    $scope.loadAll();
-                    <%_ } else { _%>
-                    $scope.reset();
-                    <%_ } _%>
-                    $('#delete<%= entityClass %>Confirmation').modal('hide');
-                    $scope.clear();
+                var modalInstance = $modal.open({
```

Comment:
You can do the same, as you did it in the user-management screen : 
 The dialog could be opened from the ui-router code, so it will have a nice url as well : /myentity/54/delete :wink: